styles.css

```diff
@@ -0,0 +1,316 @@
+    0 1px 3px rgb(0 0 0 / 30%), 
+    0 3px 5px rgb(0 0 0 / 20%), 
+    0 5px 10px rgb(0 0 0 / 25%), 
+    0 10px 10px rgb(0 0 0 / 20%), 
+    0 20px 20px rgb(0 0 0 / 15%);
+}
+.border{
+    border-right-style: groove;
+    position: relative;
+    background: linear-gradient(0deg, #000, #262626);
+}
+
+.border::before,
+.border::after {
+    content: '';
+    position: absolute;
+    background: linear-gradient(80deg, #fb0094, #0000ff, #00ff00, #ffff00, #ff0000, #fb0094, #0000ff, #00ff00, #ffff00, #ff0000);
+    background-size: 400%;
+    width: calc(100% + 4px);
+    height: calc(100% + 4px);
+    animation: rotate 3s infinite linear;
+}
+
+.border::after {
+    filter: blur(20px);   
+}
+
+.skills {
+    display: flex;    
+    justify-content: space-between;
+    margin: 1em;
+    padding: 0 20px;
+    line-height: 40px;
+    color: white;
+}
+
+.html {width: 85%; background-color: #4CAF50;} 
+.css {width: 80%; background-color: #2196F3;} 
+.js {width:70%; background-color: #f44336;} 
+.react {width: 65%; background-color: #808080;} 
+
+
+@keyframes rotate {
+    from {
+        background-position: -3000px;
+    }
+    to { 
+        background-position: 0px;
+    }
+}
+
+@keyframes pulsate {
+    50% { color: #fff; text-shadow: 0 -1px #0000004d, 0 0 5px #ffd, 0 0 8px #fff; }
+}
+
+/* ////////////////////////Slider/////////////////////// */
+.container_sidebar {
+    max-width: 33em;
+    margin: 5px;
+    position: relative;
+    overflow: hidden;
+    width: 45vw;
+    height: 45vh;
+}
+
+.sidebar {
+    height: 100%;
+    width: 35%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    transition: transform 0.5s ease-in-out;
+}
+
+.sidebar > div {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+}
+
+.sidebar h1 {
+    font-size: 40px;
+    margin-bottom: 10px;
+    margin-top: -30px;
+    text-align: center;
+}
+
+.main-slide {
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 35%;
+    width: 65%;
+    transition: transform 0.5s ease-in-out;
+}
+
+.main-slide > div {
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    height: 100%;
+    width: 100%;
+}
+
+button {
+    background-color: #fff;
+    border: none;
+    color: #aaa;
+    cursor: pointer;
+    font-size: 16px;
+    padding: 15px;
+}
+
+button:hover {
+    color: #222;
+}
+
+button:focus {
+    outline: none;
+}
+
+.container .controls button {
+    position: absolute;
+    left: 35%;
+    top: 50%;
+    z-index: 100;
+}
+
+.container .controls .down-button {
+    transform: translateX(-100%);
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+}
+
+.container .controls .up-button {
+    transform: translateY(-100%);
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+@media screen and (min-width: 400px) and (max-width: 680px){
+
```

Comment:
you need to use breakpoints (320-768, 768-992, 992+)
https://www.w3schools.com/css/css_rwd_mediaqueries.asp